src/components/Button/index.tsx

```diff
@@ -12,7 +12,17 @@ type Props = {
 } & ButtonHTMLAttributes<HTMLButtonElement>;
 
 export default function Button(props: Props): ReactElement {
-  const { className, btnType = '', children, onClick, disabled, loading, ...btnProps } = props;
+  const {
+    className,
+    btnType = '',
+    children,
+    onClick,
+    disabled,
+    loading,
+    small,
+    // Must select all non-button props here otherwise react-dom will show warning
```

Comment:
Just curious,
What was the error exactly?
Do you know if it possible to ignore that error with some react config?

Reply:
so if you are passing props to native html elements, you must only pass what's allowed [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#attributes)

the warning simple says `You are trying to pass a non-button props to a button element`. I dont know if there is a config that would ignore it, but we only do this do a few native elements (see `<Button>`, `<Input>`, `<TextArea>`, etc..)